### PR TITLE
Features/send all

### PR DESCRIPTION
**Pull request by rcmorano**
Opening this one from a local branch in favor of #52 from elribo's remote one. 

With this (working one!) I tried to map `send-all` UI with what we do in the backend with `cardano-cli` for the sake of intelligibility. But I completely broke the fee preview and "Total ADA" view on the UI :sweat_smile: 

PS: be careful to test this on wallets that DO NOT hold any other token! It took me a while to figure out why I was getting "value not conserved" lol

### CODE REVIEW

File: src/core/cardano-cli.js [hunk 1]
```diff
@@ -33,19 +33,26 @@ export function buildTxIn(addressUtxos, amount, fee) {
...
     {
         totalUsed += parseInt(txIn.value)
         tx += ` --tx-in ${txIn.txHash}#${txIn.index}`;
     }
-    tx += ` --tx-out ${toAddress}+${parseInt(amount)}`;
-    let change = parseInt(totalUsed) - parseInt(amount) - parseInt(fee);
-    if(change < 0) change = 0;
-    tx += ` --tx-out ${changeAddress}+${change}`;
+
+    if (isSendAll) {
```
Comment (rcmorano) [inline src/core/cardano-cli.js:47]: TL;DR: if we are sendingAll, we don't need any change address, right?

File: src/components/wallet/WalletDetails.vue [hunk 2]
```diff
@@ -467,8 +467,13 @@
         getFee() {
             if(this.sendForm.address.length > 0)
             {
-                const amount = (this.sendForm.amount < 1000000) ? 1000000 : this.sendForm.amount;
-                ipcRenderer.send('req:get-fee', { network: 'testnet', wallet: this.walletId, address: this.sendForm.address, amount: amount});
+                let amount;
+                if ( this.sendForm.sendAll ) {
+                  amount = this.wallet.balance;
+                } else {
+                  amount = (this.sendForm.amount < 1000000) ? 1000000 : this.sendForm.amount;
```
Comment (nothingalike) [inline src/components/wallet/WalletDetails.vue:474]: @rcmorano thinking about this particular action again... should we just set it to 1ADA or just throw validation message? just thinking whats best for user experience

Comment (nothingalike) [reply src/components/wallet/WalletDetails.vue:474]: I also don't think this question should hold up PR

**Comment by nothingalike** [comment]
I'm going to merge. My comment shouldn't hold up this PR

**Pull request merged**